packages/wallet-core/package.json

```diff
@@ -23,7 +27,7 @@
   },
   "license": "MIT",
   "scripts": {
-    "prepare": "yarn build",
+    "prepare": "rm -rf lib; yarn build",
```

Comment:
Just curious, why do we need to remove the `lib` folder?

Reply:
I don't think it's guaranteed that running the compiler will remove old, compiled files. When preparing, a precursor to publishing, we want to make sure the contents of lib are exactly what we intended.